packages/server/src/modules/app/index.ts

```diff
@@ -91,6 +92,7 @@ application.test.start = async () => {
   app.use(application.middleware.errorHandler);
 
   application.test.request = new Request();
+  application.test.fixtures = { coaches };
```

Comment:
Coach fixtures should be defined from the user model

```
application.test.fixtures.coaches = coaches;
```

Reply:
If I assign it from the user module then I get `fixtures not defined` that's why I put it there.

Or you mean the models package?

Reply:
This is how you crate a nested namespace. With nested namespace you won't get fixtures not defined.

```
// example from web
const state = createNamespace({
  actions: createNamespace({}),
  selectors: createNamespace({}),
}) as State;
```